packages/@react-native-windows/cli/src/generator-common/index.ts

```diff
@@ -43,6 +43,22 @@ export function resolveContents(
 ): string {
   let content = fs.readFileSync(srcPath, 'utf8');
 
+  if (content.includes('\r\n')) {
+    // CRLF file, make sure multiline replacements are also CRLF
+    for (const key of Object.keys(replacements)) {
+      if (typeof replacements[key] === 'string') {
+        replacements[key] = replacements[key].replace(/([^\r]?)\n/g, '$1\r\n');
```

Comment:
This wouldn't do quite the right thing for the case of multiple new lines ("/n/n" would become "/n/r/n").

I think this is correct with look behind. This is what I ended up doing for CRLF normalization in override tooling e.g. "replace(/(?<!\r)\n/g, '\r\n')"

Reply:
UTs are nice for these sorts of easy input to output algorithms (e.g. line space normalization). I had the exact same issue for my own implementation before adding tests for the interesting cases 🙂.

Reply:
Super quick example you can run on a console (using visible chars instead of escape chars)
![image](https://user-images.githubusercontent.com/835219/102553184-523dcd80-4077-11eb-9efa-990c46b275ea.png)